### PR TITLE
[WIP] Notification drawer bottom borders inconsistency

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Fix issue #2298 

@jwforres

### CODE REVIEW

File: app/styles/_notifications.less [hunk 1]
```diff
@@ -37,6 +39,9 @@ notification-drawer-wrapper {
       display: block !important;
     }
   }
+  .drawer-pf-action {
```
Comment (jwforres) [inline app/styles/_notifications.less]: this seems like a change that should be made upstream, no?

File: app/views/directives/notifications/notification-body.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <div
   class="drawer-pf-notification-inner"
-  tabindex="0"
-  ng-click="$ctrl.customScope.markRead(notification)">
+  ng-click="notification.unread && $ctrl.customScope.markRead(notification)">
```
Comment (jwforres) [inline app/views/directives/notifications/notification-body.html]: extra commit in here from your other bug

File: app/styles/_notifications.less (hunk 1 above)
Comment (benjaminapetersen) [reply app/styles/_notifications.less]: Yup, agree.  Looking into opening up a PR that way, then can either drop or close this.

File: app/views/directives/notifications/notification-body.html (hunk 2 above)
Comment (benjaminapetersen) [reply app/views/directives/notifications/notification-body.html]: Crossed the streams...

**Comment by benjaminapetersen** [comment]
Adding `[WIP]` here as well to keep from merging.  Upstream should cover it, then can close.

**Comment by benjaminapetersen** [comment]
PR opened & merged upstream here https://github.com/patternfly/patternfly/pull/788

**Comment by benjaminapetersen** [comment]
v3.29.5 & v3.29.6 https://github.com/patternfly/patternfly/releases are already available, and it looks like we have updated to the .6.  Closing.

**Pull request closed without merging**